Formula/basex.rb

```diff
@@ -5,6 +5,11 @@ class Basex < Formula
   version "8.6.6"
   sha256 "a41a6cc365741b8ee796ad22ce4acbe9f319059c5bca08fd094a351db9369acf"
 
+  devel do
+    url "http://files.basex.org/releases/latest/BaseX867-20170824.195627.zip"
```

Comment:
This is going to mess up the numbering, because brew doesn't like decreasing version number, so if the next release is e.g. 8.7.0 it will complain that this is smaller than previous devel (which is 20170824.195627).

Can we find a clever `version` to add to the `devel` block, so that it remains below the next release?

Reply:
Hi @fxcoudert, thanks for pointing out the problem.

maybe we could do it like in [aspell.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/aspell.rb#L18)?

They use the next `version` number concatenated with `-` and the `YYYYMMDD` string.

Instead of just the `-` I would add an `-rc` flag to indicate that it is a *release candidate*.
So in our case that would result in:

```
  url "http://files.basex.org/releases/latest/BaseX867-20170824.195627.zip"
  version "8.6.7-rc20170824.195627"
```

Do you think that would work out?